controllers/databaseclusterrestore_controller.go

```diff
@@ -313,6 +317,11 @@ func (r *DatabaseClusterRestoreReconciler) restorePXC(ctx context.Context, resto
 					Region:            backupStorage.Spec.Region,
 					EndpointURL:       backupStorage.Spec.EndpointURL,
 				}
+			case everestv1alpha1.BackupStorageTypeAzure:
```

Comment:
What about support of GCS?

Reply:
Per our discussion, PXC does not have GCS support yet.